engine/src/test/java/org/hibernate/search/test/dsl/SortDSLTest.java

```diff
@@ -673,7 +682,7 @@ public void set(String name, Object value, Document document, LuceneOptions luce
 				return;
 			}
 
-			luceneOptions.addNumericFieldToDocument( name, doubleValue, document );
+			document.add( new DoubleDocValuesField( name, doubleValue ) );
```

Comment:
Silly me. I think I would prefer to keep the original field in addition with the sortable one. It's more in line with what we do elsewhere. WDYT?

About adding something in LuceneOptions, I wonder if it's worth it, given the FieldBridge 2.0 plan but I agree it's a bit harsh for the user to have to go in this sort of details while we usually hide all that machinery from him. If I had to do something, I think I would add a method to add a sortable field which adds the original field and the doc values one in one go and hide this logic from the user. Of course, it is a bit redundant with the fact that we declare the field sortable a few lines above but I don't think we can do better given the current API. Anyway, I agree this part is for another day!